sympy/plotting/plot.py

```diff
@@ -820,8 +820,7 @@ def get_meshes(self):
 class ContourSeries(BaseSeries):
     """Representation for a contour plot."""
     #The code is mostly repetition of SurfaceOver2DRange.
-    #XXX: Presently not used in any of those functions.
-    #XXX: Add contour plot and use this seties.
+    #XXX: Presently used in contour_plot function
```

Comment:
Maybe you can remove the XXX, since there all the TODOs are done?

Reply:
@certik  Added the changes. Thanks for the input.